stl/inc/algorithm

```diff
@@ -179,34 +179,6 @@ namespace ranges {
         }
     };
 
-    template <class _Result, class _Wrapped, class _Unwrapped>
-    _NODISCARD constexpr _Result _Rewrap_subrange(_Wrapped& _Val, subrange<_Unwrapped>&& _UResult) {

```

Comment:
I wondered why this was moving to `<xutility>` until I saw that it's already being used there:

https://github.com/microsoft/STL/blob/ee5a216b516ef033d9153b324842369eee7653d1/stl/inc/xutility#L5674

No code change requested, but the PR description should mention this.